### PR TITLE
Add approval button for suggested scholarships

### LINKED ISSUE #16: Differentiate pending and approved scholarships
Needed is either a new column in the `/all` page which shows whether the scholarship is pending or approved. Either that or not showing pending scholarships in the `/all` page.

I think adding a column makes more sense with the current setup, but I actually think UX-wise, it might make more sense not to show pending Scholarships in the main list until they've been approved. That way spam, for example, doesn't show up with the legit scholarships.

### LINKED ISSUE #17: Wording change on empty suggested scholarships page
https://github.com/KeetonMartin/swe-project-team-3/blob/e13f25dc7aa6f03fdf1b407d2d8af4efb82ab62d/scholarshipr/index.js#L156

I think this text should say "There are no **pending** scholarships"

### PR DESCRIPTION
**Pull request by ikhovryak**
Admin can now approve the scholarship using the Approve button both from the Suggested Scholarships page and from an individual scholarship page. Also, pending scholarships no longer show up in the public list of all scholarships.

Suggested page:
image

Individual scholarship page:
image

### CODE REVIEW

File: scholarshipr/index.js [hunk 1]
```diff
@@ -364,15 +352,17 @@ function getCardHTML(scholarship) {
 	returnableString += "</li>    <li class=\"list-group-item\">";
 	returnableString += "GPA Requirement: " + gpaRequirement;
 	returnableString += "</li> </ul>  <div class=\"mx-auto\"><div class=\"card-body\">";
+	returnableString += '<td>' + 
+						` <a class="btn btn-success btn-sm mr-1" href="#" onclick="if (confirm('Approve scholarship &quot;` + scholarship.name + `&quot;?')) { window.location = '/approve?_id=` + scholarship.id + `' }">Approve</a>` +
```
Comment (qubist) [inline scholarshipr/index.js]: I think this should only show up if the scholarship is pending. Right now there's an "Approve" button on the detail view of even approved scholarships.

**Review by qubist — CHANGES_REQUESTED** [review]
One thing I noticed commented below. I think that should be changed. Otherwise looks amazing :)

File: scholarshipr/index.js (hunk 1 above)
Comment (ikhovryak) [reply scholarshipr/index.js]: Thanks, will fix it!

**Comment by ikhovryak** [comment]
@qubist, pushed a commit that would only show "Approve" button for pending scholarships detail view

**Comment by qubist** [comment]
Works perfect. I'm gonna merge!!

**Review by qubist — APPROVED** [review]

**Pull request merged**